tests/test_multistore_file.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for oauth2client.multistore_file."""
+
+import errno
+import os
+import tempfile
+import unittest
+
+from oauth2client import multistore_file
+
+
+class MultistoreFileTests(unittest.TestCase):
+
+    def test_lock_file_raises_ioerror(self):
+
+        filehandle, filename = tempfile.mkstemp()
+        os.close(filehandle)
+
+        try:
+            for error_code in (errno.EDEADLK, errno.ENOSYS, errno.ENOLCK):
+
+                class MockLockedFile(object):
```

Comment:
Because the mock's behavior is specific to this test, and I'm not intending to write a generic mock for all of multistore_file.  In we add more tests in a future CL, it can be abstracted, but I'm not in favor of trying to make it generic out of the gate.

That being said, it should exist outside of the try block here; originally I implemented just using the outer error_code.